examples/speechcommand_v1/s0/path.sh

```diff
@@ -0,0 +1 @@
+../../hi_xiaowen/s0/path.sh
```

Comment:
This should be a plain file instead of a soft link in case that we have different path for different datasets.

Reply:
okay